L/LLVM/common.jl

```diff
@@ -162,6 +170,10 @@ CMAKE_C_FLAGS=()
 
 CMAKE_FLAGS=()
 
+if [[ "${target}" != *-apple-darwin* ]]; then
```

Comment:
we should disable this on windows as well right?
```suggestion
if [[ "${target}" != *-apple-darwin* || ${target} == *mingw* ]]; then
```

Reply:
No, it's fine there too. `lld` already supports this (in fact, it likely cannot turn it off), I'm just a fool who cannot copy-paste correctly, and apparently only lld cares enough to warn me.